duckduckgo/Casper
# Remove Q&A article dates

Changes the check on the slug, to a check on the post tag.

## Testing Steps
- Checkout branch
- `gulp`
- Visit http://localhost:2368/is-duckduckgo-a-good-search-engine/
    - Confirm there is **no date** on the post
- Check any other article that is **not** a "DuckDuckGo Q&A"
    - Confirm there is a date on the post
